corehq/tabs/uitab.py

```diff
@@ -5,11 +5,13 @@
 from corehq.tabs.utils import sidebar_to_dropdown
 from dimagi.utils.decorators.memoized import memoized
 from dimagi.utils.django.cache import make_template_fragment_key
+from dimagi.utils.web import get_url_base
 from django.conf import settings
 
 
 def url_is_location_safe(url):
     from corehq.apps.locations.permissions import is_location_safe
+    url = url.split(get_url_base())[-1] if url else None
```

Comment:
Some links (including reports) use a full url like `http://localhost:8000/a/esoergel/reports/submit_history/` instead of `/a/esoergel/reports/submit_history/`, but `resolve` only accepts the latter form.